projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -198,18 +197,14 @@ export const JetpackContactFormEdit = forwardRef(
 					) }
 
 					{ 'redirect' === customThankyou && (
-						<BaseControl
```

Comment:
`URLInput` is already wrapped with [BaseControl](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/url-input/index.js#L475). Unfortunately the `div` wrapper is still needed because we can't pass a `ref` to `URLInput`'s Popover` to anchor properly.